src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
+@EntityScan("wolox.training.models")
```

Comment:
I think this annotation is redundant, the springboot autoconfiguration should read the entity from the main package